docs/anonymizer/index.md

```diff
@@ -1,22 +1,191 @@
-# Presidio Analyzer
+# Presidio Anonymizer
 
 ## Description
 
-## Table of contents
+The Presidio anonymizer is a Python based module for anonymizing detected PII text
+entities with desired values.
+
+Persidio anonymizer comes with a predefined anonymizers but can easily be extended.
 
 ## Installation
 
+=== "Using pip"
+
+    !!! note "Note"
+        Consider installing the Presidio python packages on a virtual environment like venv or conda.
+    
+    To get started with Presidio-anonymizer, run:
+    
+    ```sh
+    pip install presidio-anonymizer
+    ```
+
+=== "Using Docker"
+
+    !!! note "Note"
+        This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+    
+    ```sh
+    # Download image from Dockerhub
+    docker pull mcr.microsoft.com/presidio-anonymizer
+    
+    # Run the container with the default port
+    docker run -d -p 5001:5001 mcr.microsoft.com/presidio-anonymizer:latest
+    ```
+
+=== "From source"
+
+    First, clone the Presidio repo. [See here for instructions](../installation.md#install-from-source).
+    
+    Then, build the presidio-anonymizer container:
+    
+    ```sh
+    cd presidio-anonnymizer
+    docker build . -t presidio/presidio-anonymizer
+    ```
+
 ## Getting started
 
-- Running in Python.
-- Running using Docker.
-- Creating new anonymizers.
-- API reference.
+Presidio anonymizer comes by default with the following anonymizers:
```

Comment:
```suggestion
### Pre Configured Anonymizers

Presidio anonymizer comes by default with the following anonymizers:
```

Reply:
On design.md they're called built-in anonymizer. Consider keeping the same naming.

Reply:
Changed to built-in anonymizer